src/main/resources/jenkins/plugins/publish_over_ssh/BapSshHostConfiguration/config_no_BV.properties

```diff
@@ -27,6 +27,7 @@ password=P*s*p*r*s* / P*s*w*r*
 keyPath=P*t* t* k*y
 key=K*y
 disableExec=D*s*b*e e*e*
+jumpHost=j*m* *o*t
```

Comment:
I have absolutely no clue what this file is! Just seemed to me that every 2nd (non-space) letter is missing?! :) Should I remove?